db/schema.rb

```diff
@@ -10,11 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_02_19_145448) do
+ActiveRecord::Schema.define(version: 2021_02_25_165034) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "allocation_requests", force: :cascade do |t|
```

Comment:
Should this be in this PR?

Reply:
good catch. thought I fixed these but they've come back to haunt me. schema should now reflect only changes in this PR